delfin/tests/unit/drivers/hitachi/hnas/test_hnas.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2021 The SODA Authors.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from unittest import TestCase, mock
+
+import paramiko
+
+from delfin.tests.unit.drivers.hitachi.hnas import test_constans
+from delfin import context
+from delfin.drivers.hitachi.hnas.hds_nas import HitachiHNasDriver
+from delfin.drivers.utils.ssh_client import SSHPool
+
+
+class Request:
+    def __init__(self):
+        self.environ = {'delfin.context': context.RequestContext()}
+        pass
+
+
+class TestHitachiHNasDriver(TestCase):
+    SSHPool.get = mock.Mock(
+        return_value={paramiko.SSHClient()})
+
+    SSHPool.do_exec_command = mock.Mock(
```

Comment:
Every use case has `SSHPool.do_exec_command = mock.Mock ...`, I don't understand the meaning of this class variable, please confirm it.

Reply:
The purpose of the mock SSH is to call the interface and return the specified result